examples/react/src/App.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import "./App.css";
+const argon2 = require('argon2-browser')
+
+String.prototype.hexEncode = function(){
```

Comment:
Bad practice: don't extend prototypes, delete this code from the example. The example also doesn't need to encode as hex at all, so let's remove this function.

Reply:
done.